docs/fsspec.rst

```diff
@@ -0,0 +1,117 @@
+fsspec filesystem
```

Comment:
Can we add this paragraph into README as well please?

Reply:
May be even mention about this option at the top of the README

Reply:
Done.